tests/acl/test_acl.py

```diff
@@ -551,17 +569,18 @@ def expected_mask_routed_packet(self, pkt, ip_version):
         """Generate the expected mask for a routed packet."""
         exp_pkt = pkt.copy()
 
-        if ip_version == "ipv4":
-            exp_pkt["IP"].ttl -= 1
-        else:
-            exp_pkt["IPv6"].hlim -= 1
-
         exp_pkt = mask.Mask(exp_pkt)
         exp_pkt.set_do_not_care_scapy(packet.Ether, "dst")
         exp_pkt.set_do_not_care_scapy(packet.Ether, "src")
 
         if ip_version == "ipv4":
             exp_pkt.set_do_not_care_scapy(packet.IP, "chksum")
+            # In multi-asic we cannot determine this so ignore.
+            exp_pkt.set_do_not_care_scapy(packet.IP, 'ttl')
+        else:
+            # In multi-asic we cannot determine this so ignore.
+            exp_pkt.set_do_not_care_scapy(packet.IPv6, 'hlim')
```

Comment:
Why skip checking ttl and hlim?

Reply:
@wangxin for multi-asic packet can get forwarded/routed from same asic (incoming ttl-1) or via back-end asic (incoming ttl -3) so we can not predict what path packet will traverse so ignoring it.